x-pack/plugins/osquery/public/packs/queries/ecs_mapping_editor_field.tsx

```diff
@@ -577,7 +539,10 @@ const OsqueryColumnFieldComponent: React.FC<OsqueryColumnFieldProps> = ({
           {Prepend}
         </EuiFlexItem>
         <EuiFlexItem css={overflowCss}>
-          <ResultComboBox
+          {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+          {/* @ts-ignore*/}
```

Comment:
Another ts-ignore you can probably lose now
```suggestion
```

Reply:
@tomsonpl I did another pass on the code changes and they look amazing, thanks for refactoring to use so many EUI tokens!! ❤️ 

It looks like this comment was missed/still has a ts-ignore just FYI - this is my last comment! :)

Reply:
Thanks for catching this. Unfortunately this one has some issues with it's options types and that is cumbersome to fix. Since it was just my onweek project I do not want to spend too much time on this thus fixing this type has to wait ;(

Reply:
Gotcha, thanks for clarifying! :)